### ISSUE TITLE
Finish migration to Heroku Cedar

### ISSUE THREAD

**Issue by chrishunt** [chrishunt/huntxmas#8]
See the Heroku [migration instructions](https://devcenter.heroku.com/articles/cedar-migration).

After #7 is done, we need to make the [cedar app live](https://devcenter.heroku.com/articles/cedar-migration#make-cedar-app-live) by
- Restoring a pg backup from the older bamboo stack
- Clear/set the domain settings for the new cedar app
- Point DNS at the new cedar version
- Destroy the bamboo version

**Comment by chrishunt** [comment]
Closing because this is super old, not used anymore, and it's already done. :wave:
